### DISCUSSION TITLE
I have no idea how to use these proxies-

### DISCUSSION THREAD

**Discussion by Shadr0w** [3kh0/ext-remover#690] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have no idea how to activate Point Blank. Can someone help me?

I feel stupid tbh .

**Comment by op-max** [comment, marked as answer]
Did you not read the read.md or the title or anything?

These are exploits meant to disable or eliminate school blocking systems.
Also, point blank is a blookmarket so umm- there are many tutorials on that.

**Comment by Sereneplays08** [comment]
i tried ltmeat idk how to use any of the others i just know ltmeat works
